setup.py

```diff
@@ -92,14 +92,13 @@ def read_requirements(filename):
     url="https://github.com/aws/sagemaker-python-sdk/",
     license="Apache License 2.0",
     keywords="ML Amazon AWS AI Tensorflow MXNet",
-    python_requires=">= 3.6",
+    python_requires=">= 3.7",
```

Comment:
This should be 3.8 right?

Reply:
If we remove py37 from buildspec but docker image still contains py37 installation, setting to 3.8 will cause unit tests to fail. Leaving like this until I can update the docker image